spec/test.spec.js

```diff
@@ -316,84 +316,89 @@ describe('S3Adapter tests', () => {
   });
 
   describe('getFileLocation', () => {
-    const testConfig = {
-      mount: 'http://my.server.com/parse',
-      applicationId: 'xxxx',
-    };
-    let options;
-
-    beforeEach(() => {
-      options = {
-        directAccess: true,
-        bucketPrefix: 'foo/bar/',
-        baseUrl: 'http://example.com/files',
+    ['http://example.com/files', () => 'http://example.com/files'].forEach((baseUrl) => {
```

Comment:
Could you please refactor this for simplicity. We normally do not use such a pattern anywhere where we wrap tests into a `forEach`.

Reply:
@danielsanfr Do you think you have time to do this minor refactor? Then we can merge this feature and make is available officially.